app/controllers/api/encryptables_controller.rb

```diff
@@ -110,7 +110,8 @@ def query_param
   end
 
   def encryptable_move_handler
-    EncryptableMoveHandler.new(entry, users_private_key, current_user)
+    entry.sender_id = nil
+    EncryptableMoveHandler.new(entry, session[:private_key], current_user)
```

Comment:
Removed, because users_private_key needs `Authorization-Password` in request headers and we don't add them here. This method is working and is not bad, I think :smile: